export function throwsError() {
    throw new Error('Custom Error thrown');
}
